cfme/configure/configuration/__init__.py

```diff
@@ -1156,7 +1134,7 @@ def select_by_name(name):
                     "Schedule '{}' could not be found for selection!".format(name)
                 )
 
-        sel.force_navigate("cfg_settings_schedules")
+        navigate_to(cls, 'All')
```

Comment:
Frankly speaking, I didn't want to do any refactoring of existing code because more changes -> more possible errors. 
However, you can convince event dead :).
Removed all class methods and added simple object methods.